packages/web/src/pages/upload-page/fields/MultiTrackSidebar.tsx

```diff
@@ -0,0 +1,31 @@
+import { HarmonyButton } from '@audius/stems'
+import cn from 'classnames'
+
+import layoutStyles from 'components/layout/layout.module.css'
+
+type MultiTrackSidebarProps = {
+  index: number
+  limit: number
+  setIndex: (index: number) => void
+}
+
+export const MultiTrackSidebar = (props: MultiTrackSidebarProps) => {
+  const { index, setIndex, limit } = props
```

Comment:
just something to consider long term, if we can try to understand what the best api would be for this element. do we use `onChange` as the prop instead of setIndex? or maybe `onNext` `onPrevious`? basically trying to be as clear as we can with what the expected usage is for a component, and minimize the number of actions a component can allow. this is all more philosophical, so curious your thoughts.

Reply:
my thinking is setIndex or an onChange will be correct because we select by index in the sidebar, and eventually the next/prev buttons move to the bottom of the form